packages/react-components/src/organisms/UserProfileBackground.tsx

```diff
@@ -29,10 +29,23 @@ const linksContainer = css({
   },
 });
 
+const noMarginTextStyle = css({
```

Comment:
This intrudes into styling of other components and should not be necessary given the headline and paragraph spacings are defined by the design system, can you please push back on this? I think the most we should offer if it really needs to be less space between is using only the headline margin (by replacing the `Paragraph` with a plain `p` that does not have margin)

Reply:
Agreed. Discussed this with James and he agreed on the final result